data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -162,6 +162,9 @@ then
 		--volume "$PWD:/assets:z" \
 		"${CONFIG_OPERATOR_IMAGE}" \
 		/usr/bin/cluster-config-operator render \
+		--cloud-provider-config-input-file=/assets/manifests/cloud-provider-config.yaml \
+		--cluster-infrastructure-input-file=/assets/manifests/cluster-infrastructure-02-config.yml \
+		--cloud-provider-config-output-file=/assets/config-bootstrap/cloud-provider-config-generated.yaml \
```

Comment:
```suggestion
		--cloud-provider-config-output-file=/assets/config-bootstrap/manifests/cloud-provider-config-generated.yaml \
```
I think that will make sure other operators like MCO can use it later on and also the file will be pushed to api.
see https://github.com/openshift/installer/pull/3831/files#diff-ce82c1d8a44f7dfc41dfc024085ccfeeR172
```
cp config-bootstrap/manifests/* manifests/
```